python/doc/pyplots/LinearModelValidation.py

```diff
@@ -0,0 +1,21 @@
+import openturns as ot
+from openturns.viewer import View
+
+func = ot.SymbolicFunction(
+    ['x1', 'x2', 'x3'],
+    ['x1 + x2 + sin(x2 * 2 * pi_) / 5 + 1e-3 * x3^2']
+)
+
+dimension = 3
+
+distribution = ot.JointDistribution([ot.Normal()] * dimension)
+sampleSize = 50
+inputSample = distribution.getSample(sampleSize)
+outputSample = func(inputSample)
+algo = ot.LinearModelAlgorithm(inputSample, outputSample)
+algo.run()
+result = algo.getResult()
+splitter = ot.LeaveOneOutSplitter(sampleSize)
+validation = ot.LinearModelValidation(result, splitter)
```

Comment:
it is in experimental!

Reply:
I guess it is still not fixed, isn't it?

Reply:
Indeed, it is not fixed. As soon as I involve 29d054f59f77978b93f739d942d151d8c056c94c in the PR, the error comes back again.

Reply:
That's strange...